va/Makefile.am

```diff
@@ -55,6 +56,7 @@ libva_source_h = \
 	va_fei_h264.h		\
 	va_enc_mpeg2.h		\
 	va_enc_vp9.h            \
+	va_str.h		\
```

Comment:
Do you want to install this header?  Is it going to be used by applications? The reset looks good.

Reply:
Yes, this will be used by applications... vainfo will be the first app to use it. (01org/libva-utils/pull/61)

Reply:
I agree, unit tests would be another good place to use it 👍

Reply:
if that is going to be done on a future patch, please file an issue before this gets merged.  (trying to keep track of it).

Reply:
@chivakker Done... see references below